backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5585670_sys_gh10990_AskForOrgChange_SysConfig.sql

```diff
@@ -0,0 +1,7 @@
+-- 2021-04-15T12:56:01.539Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value) 
+VALUES (0,0,541364,'S',TO_TIMESTAMP('2021-04-15 15:56:01','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','AskForOrgChangeOnRegionChange',
```

Comment:
please add a description, so one understand what happens if this is set to Y

Reply:
Added